src/main/kotlin/no/nav/familie/ef/sak/service/BehandlingService.kt

```diff
@@ -105,4 +112,9 @@ class BehandlingService(private val søknadRepository: SøknadRepository,
         }
     }
 
+    fun oppdaterJournalpostIdPåBehandling(journalpostId: String, behandling: Behandling){
+        behandling.journalposter = behandling.journalposter + BehandlingJournalpost(journalpostId = journalpostId, sporbar = Sporbar() )
```

Comment:
Lurer på om denne typen av oppdateringer trenger en lock av behandlingen for å unngå slitsomme situasjoner? :P